wrappers/rust/icicle-core/src/msm/tests.rs

```diff
@@ -25,6 +24,7 @@ fn generate_random_affine_points_with_zeroes<C: Curve>(size: usize, num_zeroes:
     points
 }
 
+// TODO: this should be INCORRECT for more than 1 device!! correct it
```

Comment:
?

Reply:
Sorry forgot to remove. It does pass on multi-device machines